lib/entry/FlavorDownloader.js

```diff
@@ -312,10 +316,19 @@ var FlavorDownloader = (function() {
         var that = this;
         that.logger.debug("Stopping flavor downloader");
         that.shouldRun = false;
+        var fullPath = that.destFolderPath + '/' + that.newPlaylistName;
```

Comment:
no need to put  that.newPlaylistName into this, you have a refrene to the CTOR scope, just use  newPlaylistName